PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1196,6 +1196,18 @@ Do you want to launch Notepad++ in Administrator mode?"/>
             <finder-select-all value="Select all"/>
             <finder-clear-all value="Clear all"/>
             <finder-open-all value="Open all"/>
+            <common-ok value="OK"/>
+            <common-cancel value="Cancel"/>
```

Comment:
`OK` and `Cancel` shouldn't be localized since it's the system takes charge in them.

Reply:
Hmm... never mind, I was wrong.

Reply:
Yes, it's pretty good common-ok & common-cancel.

Reply:
> Hmm... never mind, I was wrong.

No Problem. I have resolved conflict now.